### ISSUE TITLE
[Request] Do not translate if string starts with "

### ISSUE THREAD

**Issue by xeroc** [screeninteraction/jekyll-multiple-languages-plugin#44]
My page is multi-language and I use you plugin (which is great so far).
However, when prototyping parts of the page (mainly still only in english) I use constructs like those:

    title: "Security and Control over Your Money" 

jekyll attempts to translate the string, but I'd like to only translate those strings that do not start with ", like

    title: security.control

Would this be possible?

**Comment by kurtsson** [comment]
Yes, just add an if-case in for strings starting with ". But Im not sure I think this is a good design since you should never have untranslated strings and it might be bad to make this easier.
